pkg/libmxnet.pkg

```diff
@@ -21,6 +21,7 @@ It has been built with OpenCV and OpenMP support disabled.
 doc_install_dir = '/usr/share/doc/' + VAR.fullname + '/'
 
 ARGS = FUN.mapfiles('./lib', '/usr/lib', libs, append_suffix=False) + [
-    'LICENSE=' + doc_install_dir + 'copyright',
+    'NOTICE=' + doc_install_dir + 'copyright',
+    'LICENSE=' + doc_install_dir,
```

Comment:
And no, I don't need to name the file on the RHS if it's the same as the original.  Just specifying the directory where it's to be placed is sufficient.